calcom/cal.com
# refactor: Team Creation Flow [CAL-2751]

## What does this PR do?



This PR changes the team creation flow. Currently you would have the option to create a team and then go through the Stripe checkout flow to publish the team.

This PR changes the flow so now, to create a team you need an active subscription before continuing.

Fixes # (issue)



https://www.loom.com/share/47e1a588a85d4e6fa47a989dee55282b

## Requirement/Documentation



- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change





## How should this be tested?



- Before checking out the branch, create a team but do not publish
- Now checkout the branch this PR is on
- Fill in the first step of creating a team
- You should be brought to the checkout page. Complete this step
- After you should be brought back to the add members page with the team now created in the DB

## Mandatory Tasks


## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
